### PR TITLE
remove .net 5. and add support for .net 7

### PR DESCRIPTION
**Pull request by vb2ae**
remove .net 5. and add support for .net 7 update global.json for version of visual studio 2022 version 17.4

Addresses issue #836 and issue #838

### CODE REVIEW

File: samples/features/Features.NetFive/Features.NetFive.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
```
Comment (KasperSK) [inline samples/features/Features.NetFive/Features.NetFive.csproj:2]: Should we rename this project to Net instead of NetFive?

Comment (vb2ae) [reply samples/features/Features.NetFive/Features.NetFive.csproj:2]: That is a good Idea I will work on that

Comment (vb2ae) [reply samples/features/Features.NetFive/Features.NetFive.csproj:2]: updated

**Review by kfordmad — COMMENTED** [review]
This seems to tick the normal checkboxes.  @KasperSK is there anything specific needed here.  I would happily work on this.

**Comment by vb2ae** [comment]
@kfordmad we just need to get the pr approved and we can merge it into the master branch

**Comment by kfordmad** [comment]
> 

Yeah.  It seems to give me the option to approve, but I suspect it wont actually go through, and I am not a member of the project, so I figured I would poke.  Was very interested in using this in a new project, and was hoping to use 7.  Suppose I can fork and self compile until this is approved.

**Review by KasperSK — APPROVED** [review]

**Pull request merged**